draft-ZIP-0226.rst

```diff
@@ -0,0 +1,303 @@
+
+In order to ensure the security of the transfers, and as we will explain below, we are replacing input dummy notes for custom assets, as we need to enforce that the type of the output note of that split action is the output of a PRF.
+
+Finally, in this ZIP we also describe the *burn* mechanism, which is a direct extension of the transfer mechanism. The burn process uses a similar mechanism than what is used in Orchard to unshield ZEC, by using the :math:`\mathsf{valueBalance}` of the asset in question. Burning assets is useful for many purposes, including bridging of wrapped assets and removing supply of native assets.
+
+Orchard Protocol Changes
+========================
+
+Most of the protocol is kept the same as the Orchard protocol released with NU5, except the following that will be deployed to mainnet in subsequent network upgrades.
+
+Asset Types
+-----------
+
+In the OSA protocol, we include a new variable, the asset type identifier, :math:`\mathsf{type}`, which is generated as a 32-byte string during issuance (as described in the Issuance ZIP [#zip-0227]_). The :math:`\mathsf{type}` will then be publicly hashed into the corresponding group, in this case the Pallas curve, by using the :math:`\mathsf{GroupHash}`
+function. In fact, every ZSA note will contain the group element representation of the asset type identifier. This will enable a much more elegant and simple version of the circuit, as we will see.
+
+We denote the string as :math:`\mathsf{type}` and we denote the equivalent group representation as :math:`\mathsf{type}_{\mathbb{G}}` when mapped into a specific group, :math:`\mathbb{G}`. Note that the type of the ZEC asset will be kept as the original value base-point, :math:`\mathcal{V}^\mathsf{Orchard}`
+
+In future network and protocol upgrades, the same asset type string can be carried on, with potentially a mapping into a different curve or group. In that case, the turnstile should know how to transform the asset type from one group to another one.
+
+Note Structure & Commitment
+---------------------------
+
+First, we need to adapt the components that define the assets, i.e.: *notes*. A ZSA note differs from an Orchard note by including the type of asset, :math:`\mathsf{type}_\mathbb{P}`. So an ZSA note looks like:
+
+
+:math:`(\mathsf{g_d, pk_d, v, \rho, \psi, type}_{\mathbb{P}})`
+
+
+Where :math:`\mathsf{type}_\mathbb{P}` is the unique random group element that identifies each asset in the Pallas curve [#protocol-pallasandvesta]_, [#pasta-evidence]_ of the Orchard protocol. 
```

Comment:
```suggestion
Where :math:`\mathsf{type}_\mathbb{P}` is the unique random element of the Pallas group [#protocol-pallasandvesta]_, that identifies each asset in the Orchard protocol. 
```

Reply:
Added these changes in [PR#649](https://github.com/zcash/zips/pull/649).